### ISSUE TITLE
Docker entrypoint broken in v1.0.0-beta.20

### ISSUE THREAD

**Issue by tomasaschan** [GoogleContainerTools/kpt#3539]
The instructions [here](https://kpt.dev/installation/kpt-cli?id=docker) don't work for 1.0.0-beta.20:

### Expected behavior
```
$ docker run gcr.io/kpt-dev/kpt:v1.0.0-beta.20 version
1.0.0-beta.20
```
### Actual behavior
```
$ docker run gcr.io/kpt-dev/kpt:v1.0.0-beta.20 version
exec /usr/local/bin/kpt: no such file or directory
```

It seems to work as expected on 1.0.0-beta.19:
```
$ docker run gcr.io/kpt-dev/kpt:v1.0.0-beta.19 version
1.0.0-beta.19
```

**Comment by natasha41575** [comment]
@droot As discussed in standup, this is the same error message I got here https://github.com/GoogleContainerTools/kpt/issues/3528#issuecomment-1234631750

**Comment by tomasaschan** [comment]
@natasha41575 @droot This seems to be fixed in 1.0.0-beta.21
```sh
$ docker run gcr.io/kpt-dev/kpt:v1.0.0-beta.21 version
1.0.0-beta.21
```

**Comment by droot** [comment]
Thanks for verifying it @tomasaschan
